crates/holochain/Cargo.toml

```diff
@@ -103,8 +103,8 @@ matches = { version = "0.1.8", optional = true }
 holochain_test_wasm_common = { version = "^0.4.0-dev.6", path = "../test_utils/wasm_common", optional = true }
 kitsune_p2p_bootstrap = { version = "^0.3.0-dev.7", path = "../kitsune_p2p/bootstrap", optional = true }
 unwrap_to = { version = "0.1.0", optional = true }
-sbd-server = { version = "=0.0.3-alpha", optional = true }
-tx5-go-pion-turn = { version = "=0.0.11-alpha", optional = true }
+sbd-server = { version = "=0.0.4-alpha", optional = true }
```

Comment:
@steveej - *This* bump is covered by unit/integrations because sbd server is used in the sweetest setup:

https://github.com/holochain/holochain/blob/0e44fcd3bdac0e3b36b40f76935bea8647753f43/crates/holochain/src/sweettest/sweet_conductor_config_rendezvous.rs#L80-L87